# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: js/src/tests/jstests.list
# Commit: 1b8c42577e24
# Full Hash: 1b8c42577e2410dbbe050bb30b0ccced9d7c3771
# Author: Ashley Hauck <khyperia@mozilla.com>
# Date: 2019-03-20 10:52:37
# Regressor Bug: 1535166
# File Overlap Count: 1
# Description:
#   Bug 1535166 - Implement computed field names. r=jorendorff
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D23408
# ==============================================================================

diff -r 28370bf14b97 -r 1b8c42577e24 js/src/tests/jstests.list
--- a/js/src/tests/jstests.list	Tue Mar 19 22:02:49 2019 +0000
+++ b/js/src/tests/jstests.list	Tue Mar 19 20:23:03 2019 +0000
@@ -12,13 +12,6 @@
 
 # Fields are not fully implemented yet
 # https://bugzilla.mozilla.org/show_bug.cgi?id=1499448
-skip script non262/fields/access.js
-skip script non262/fields/basic.js
-skip script non262/fields/error.js
-skip script non262/fields/field_types.js
-skip script non262/fields/literal.js
-skip script non262/fields/mixed_methods.js
-skip script non262/fields/quirks.js
 skip script non262/reflect-parse/class-fields.js
 
 